tests/suites/test_suite_constant_time_hmac.function

```diff
@@ -78,11 +78,20 @@ void ssl_cf_hmac(int hash)
 #endif
 
     /*
-     * Test all possible lengths up to a point. The difference between
+     * Test various possible lengths up to a point. The difference between
      * max_in_len and min_in_len is at most 255, and make sure they both vary
      * by at least one block size.
+     *
+     * It is quite expensive to test all possible lengths, so we test values
+     * which are within 1 of a multiple of 64, plus loop end values.
```

Comment:
Multiples of 16 is a relevant threshold because it's the block size (in 2.28 we also support DES with an 8-byte block size). I don't see where 64 comes from.

Reply:
I can change the hard-coded 64 to `block_size` then, if we think that will get us sensible coverage?

Reply:
Yes, I think that would be good.